dashboard/db/migrate/20210119225102_copy_activity_section_title_to_progression_name.rb

```diff
@@ -0,0 +1,22 @@
+class CopyActivitySectionTitleToProgressionName < ActiveRecord::Migration[5.2]
+  def up
+    Script.where("properties -> '$.is_migrated' = ?", true).each do |script|
+      script.lessons.each do |lesson|
+        lesson.lesson_activities.each do |activity|
+          activity.activity_sections.each do |section|
+            next if section.script_levels.empty?
+            section.progression_name = section.name
+            section.save!
+            section.script_levels.each do |sl|
+              sl.progression = section.name
+              sl.save!
+            end
+          end
+        end
+      end
+    end
+  end
+
+  def down
+  end
+end
```

Comment:
this file looks like it is part of a a different chain of PRs. does the merge base just need to be updated? unfortunately it's a bit hard to see what's what from the commit history.